internal/output/diff.go

```diff
@@ -156,6 +183,16 @@ func resourceToDiff(currency string, diffResource Resource, oldResource *Resourc
 				ui.FaintString(formatCostChangeDetails(currency, oldCost, newCost)),
 			)
 		}
+		if showEmissions {
+			if oldEmissions == nil && newEmissions == nil {
+				s += "  Monthly emissions depends on usage\n"
```

Comment:
```suggestion
				s += "  Monthly emissions depend on usage\n"
```
[Non blocking] It admittedly depends on how the sentence is heard...